akka-cluster/src/multi-jvm/resources/logback-test.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="info">
+        <appender-ref ref="STDOUT" />
+    </root>
+    <logger name="io.netty.util.Recycler" level="ERROR" />
+    <logger name="io.netty.buffer.PoolThreadCache" level="ERROR" />
+</configuration>
```

Comment:
is slf4j/logback needed by Netty or what is the reason we need this now?

is this included by the other sbt projects that have multi-jvm tests or is same needed in more places?

Reply:
I see this in logs:
```
2023-08-14T08:44:19.0726537Z [0m[[0m[31merror[0m] [0m[0m[[34mJVM-2[0m] SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".[0m
2023-08-14T08:44:19.0727622Z [0m[[0m[31merror[0m] [0m[0m[[34mJVM-2[0m] SLF4J: Defaulting to no-operation (NOP) logger implementation[0m
```

Reply:
I'll add it for all multi-jvm tests then.